src/sentry/static/sentry/app/views/settings/organizationMembers/inviteRequestRow.tsx

```diff
@@ -62,11 +73,34 @@ const InviteRequestRow = ({
           </Tooltip>
         )}
       </div>
-      <div>{inviteRequest.roleName}</div>
+      <div>
```

Comment:
Are these divs necessary?

Reply:
I'll fix this up in a separate PR in a min.